contracts/src/TokenHome/ERC20TokenHome.sol

```diff
@@ -130,8 +173,8 @@ contract ERC20TokenHome is IERC20TokenHome, TokenHome {
         uint256 remainingAllowance = token.allowance(address(this), message.recipientContract);
 
         // Reset the recipient contract allowance to 0.
-        // Use of {safeApprove} is okay to reset the allowance to 0.
-        SafeERC20.safeApprove(token, message.recipientContract, 0);
+        // Use of {forceApprove} is okay to reset the allowance to 0.
+        SafeERC20.forceApprove(token, message.recipientContract, 0);
```

Comment:
```suggestion
        SafeERC20.forceApprove(token, message.recipientContract, 0);
```

Super nit: this comment was geared towards calling out the vulnerability pattern in `safeApprove` does not apply in this case, but I don't think it's relevant if not using `safeApprove` at all anymore.